cli/tests/integrations/test_ssl.py

```diff
@@ -0,0 +1,26 @@
+from dcos import http, util
+from dcos.errors import DCOSException
+
+import pytest
+from six.moves import urllib
+
+DCOS_URL = util.get_config()['core.dcos_url'].replace('http://', 'https://')
+DCOS_URL = urllib.parse.urljoin(DCOS_URL, 'marathon/')
+
+
+def test_dont_verify_ssl():
+    resp = http.request("get", DCOS_URL, verify=False)
+    assert resp.status_code == 200
+
+
+def test_verify_ssl():
+    with pytest.raises(DCOSException) as e:
+        http.request("get", DCOS_URL, verify=True)
+    assert "certificate verify failed" in str(e)
+
+
+def test_verify_ssl_with_bad_cert():
+    with pytest.raises(DCOSException) as e:
+        cert = 'tests/data/ssl/fake.pem'
+        http._request("get", DCOS_URL, verify=True, cert=cert)
```

Comment:
Same. Let's test this using the dcos cli. Also, I would like to see the full output from this user story. Ideally in the test. If not possible then a comment in the PR.